src/SimpleThings/EntityAudit/AuditConfiguration.php

```diff
@@ -124,14 +158,25 @@ public function createMetadataFactory()
         return new Metadata\MetadataFactory($this->auditedEntityClasses);
     }
 
-    public function setCurrentUsername($username)
```

Comment:
Removing `setCurrentUsername` is a BC break. It is simple to work around this by making this internally do:

```php
/**
 * @deprecated
 */
public function setCurrentUsername($username)
{
    $this->setUsernameCallable(function () use ($username) {
        return $username;
    });
}
```